### DISCUSSION TITLE
Tables with the possibility of creating a tree structure

### DISCUSSION THREAD

**Discussion by qcol** [filamentphp/filament#10022] [Ideas]
Hi

It would be great to be able to create tables with a tree structure. Something similar to here: https://github.com/solutionforest/filament-tree , with indentation but retaining the capabilities that a table has (text column, checkbox column, etc.) - full interaction with the data record as in tables. 

Or allow model extensions to be used in using Eloquent extension with getEloquentQuery() or modifyQueryUsing()?
Extensions such as https://github.com/staudenmeir/laravel-adjacency-list could then be used to generate a tree-structured database query with depth information.

**Comment by Adysone** [comment]
Hello,

DId you progress on this subject/feature?
